tests/core/snapshots/apollo-federation-entities.md_client.snap

```diff
@@ -0,0 +1,73 @@
+---
+source: tests/core/spec.rs
+expression: formatted
+---
+scalar Bytes
+
+scalar Date
+
+scalar DateTime
+
+scalar Email
+
+scalar Empty
+
+scalar Int128
+
+scalar Int16
+
+scalar Int32
+
+scalar Int64
+
+scalar Int8
+
+scalar JSON
+
+scalar PhoneNumber
+
+type Post {
+  id: Int!
+  title: String!
+}
+
+type Query {
+  """
+  Apollo federation Query._entities resolver
+  """
+  _entities(representations: [_Any!]!): [_Entity]!
+  """
+  Apollo federation Query._service resolver
+  """
+  _service: _Service!
+  user(id: Int!): User
+}
+
+scalar UInt128
+
+scalar UInt16
+
+scalar UInt32
+
+scalar UInt64
+
+scalar UInt8
+
+scalar Url
+
+type User {
```

Comment:
`@key` is missing here.

Reply:
the `_client` file contains no directives and doesn't seems like `@key` needs special treatment